### ISSUE TITLE
Explain lambda parameters

### ISSUE THREAD

**Issue by sumantha-NTS** [bcmi/Object-Shadow-Generation-Dataset-DESOBA#15]
Hi,
Please explain below parameters in SGRNet options, not able understand with the name.

lambda_GAN: 0.01                          	
lambda_GAN_mask: 0.5                           	
lambda_I1: 10.0                          	
lambda_L1: 0.0                           
lambda_M1: 10.0                          
lambda_P1: 1                             
lambda_REF1: 1                             
lambda_STN1: 1                             
lambda_TV1: 1                             
lambda_bd: 0.0
finetuning: False                         
finetuning_dir: None                          
finetuning_epoch: None                          
  finetuning_name: None

**Comment by hy-zpg** [comment]
you can only focus on: lambda_GAN(lambda_I1, lambda_M1,lambda_P1) is the hyperparameter of GAN(image reconstruction, mask reconstruction, illumination parameter regression).
